platform/admin/ui/src/main/webapp/js/views/installer/Configuration.view.js

```diff
@@ -19,28 +19,242 @@ define([
+            });
+        }
+    });
+
+
+    var systemProperties = new SystemProperties();
+    systemProperties.fetch({});
+
+
+
+/*
+* Layout
+*/
     var ConfigurationView = Marionette.Layout.extend({
         template: 'configurationTemplate',
         className: 'full-height',
+        model: systemProperties,
+        regions: {
+            configurationItems: '#config-form',
+            certificates: '#certificate-configuration'
+        },
 
         initialize: function(options) {
+
             this.navigationModel = options.navigationModel;
-            this.navigationModel.set("hidePrevious",true);
+
+            this.listenTo(this.navigationModel, 'previous', this.previous);
             this.listenTo(this.navigationModel,'next', this.next);
+
+            this.listenTo(this.model, 'invalid', function(model, error){
+                var layout = this;
+
+                this.model.each(function(m){
+                    if (model.get('key') === m.get('key')) {
+                        layout.$("[name='" + m.get('key') + "Error']").hide();
+                    }
+
+                    _.each(error, function(errorItem){
+                       if (m.get('key') === errorItem.id) {
+                        layout.$("[name='" + m.get('key') + "Error']").show().html(errorItem.message);
+                       }
+                    });
+                });
+            });
+
+            // clear error message on model validation
+            this.listenTo(this.model, 'validModel', function(model) {
+                var layout = this;
+
+                this.model.each(function(m){
+                    if (m.get('key') === model.get('key')) {
+                        layout.$("[name='" + m.get('key') + "Error']").hide();
+                    }
+                });
+            });
+
         },
         next: function() {
-            this.navigationModel.nextStep('', 100);
+            var layout = this;
+
+            // loop through models and check for hostname change, validation errors and set redirect url
+            var hostChange = true;
+            var hasErrors = false;
+
+            this.model.each(function (model) {
+                hasErrors = hasErrors || model.validationError;
+
+                if(model.get('title') === 'Host') {
+                    if(model.get('value') === model.get('defaultValue')){
+                        hostChange = false;
+                    }
+                } else if(model.get('title') === 'HTTPS Port') {
+                    layout.navigationModel.set('redirectUrl','https://localhost:'+model.get('value')+'/admin/index.html');
+                }
+            });
+
+
+            if (! hasErrors) {
+
+                var propertySave = this.model.save();
+                if(propertySave){
+                    propertySave.done(function(){
+                        if(hostChange) {
+
+                            var certSave = layout.certificates.currentView.saveCerts();
```

Comment:
If you created the certificate model that was then passed to this view, you could simply call save() on that model and have a listener in the certificates view to listen to that call rather than trying to dig down into the layout to call the view directly. Doing this in backbone is not using their api correctly and could break in a future release of backbone.